sass/abstracts/_mixin.scss

```diff
@@ -0,0 +1,68 @@
+@mixin row {
+  display: flex;
+  flex-direction: row;
+  @content;
+}
+
+@mixin column {
+  display: flex;
+  flex-direction: column;
+  @content;
+}
+@mixin icon {
+  display: flex;
+  justify-content: center;
+  border-radius: 50%;
+  border: 1px solid $color-primary;
+  background-color: $white-secondary;
+  @content;
+}
+
+@mixin input {
+  border-radius: 0.5em;
+  padding: 0.8em 0em;
+  padding-left: 0.8em;
+  width: 100%;
+  border: 1px solid $color-primary;
+  font-size: $font-size;
+  margin-bottom: 0.8em;
+  margin-right: 0.8em;
+  background-color: $white-secondary;
+  &:focus {
+    background-color: $white-primary;
+    outline: none;
+  }
+  &::-webkit-input-placeholder {
+    color: $color-secondary;
+  }
+  @content;
+}
+
+@mixin center {
+  display: flex;
+  justify-content: center;
+}
+
+@mixin btn {
```

Comment:
Big bro, can you make this a function that takes (height, weight, color and background color ) so that everyone can call if he wants to design any button?

Reply:
but still you can pass in these attributes, it is a mixin!